Patch 1

strategicalliances opened the pull request.

strategicalliances committed:
TFAT-977 (Daan)

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/mendix/docs?pullRequest=6407) <br/>All committers have signed the CLA.

strategicalliances committed:
Update amazon-bedrock.md

Corrected a typo.

strategicalliances committed:
Update amazon-bedrock.md

MarkvanMents commented:
Hi,
Thanks for the PR.
I've switched this PR to point at @katarzyna-koltun-mx 's original PR.
Is this likely to be published this week, or can it wait until Kasia is back from leave?
Feel free to push more changes to this branch if there is anything else needed.
Could you also get @mx-awsdevteam to sign the CLA so that we get a clean oversight of what work is needed.
Thanks
Mark

strategicalliances committed:
Update amazon-bedrock.md

strategicalliances committed:
Update amazon-bedrock.md

MarkvanMents commented:
@mx-awsdevteam 
Can you let me know when this PR is ready to be merged and I'll do a final review.
Thanks

strategicalliances committed:
Update amazon-bedrock.md

strategicalliances committed:
Update amazon-bedrock.md

strategicalliances committed:
Update amazon-bedrock.md

strategicalliances committed:
Update amazon-bedrock.md

katarzyna-koltun-mx committed:
Update amazon-bedrock.md

katarzyna-koltun-mx committed:
Update amazon-bedrock.md

katarzyna-koltun-mx committed:
Update amazon-bedrock.md

katarzyna-koltun-mx committed:
Update amazon-bedrock.md

katarzyna-koltun-mx committed:
Update amazon-bedrock.md

katarzyna-koltun-mx committed:
Update amazon-bedrock.md

katarzyna-koltun-mx committed:
Update amazon-bedrock.md

katarzyna-koltun-mx committed:
Update amazon-bedrock.md

strategicalliances committed:
Update amazon-bedrock.md

katarzyna-koltun-mx committed:
Update amazon-bedrock.md

katarzyna-koltun-mx committed:
Update amazon-bedrock.md

katarzyna-koltun-mx committed:
Update amazon-bedrock.md

katarzyna-koltun-mx committed:
Update amazon-bedrock.md

katarzyna-koltun-mx committed:
Update amazon-bedrock.md

katarzyna-koltun-mx commented:
The purpose of this PR is to review documentation that will first be published on the Marketplace. I will keep it open until the Marketplace record is published, then I will merge into https://github.com/mendix/docs/pull/6401/files.

strategicalliances committed:
Update amazon-bedrock.md

strategicalliances committed:
Update aws-lambda-connector.md

Changes made to 4.1 to reflect changes in the lambda connector.

strategicalliances committed:
Update aws-lambda-connector.md

Added documentation on the new "ListFunctionsRequest" entity.

strategicalliances committed:
Update aws-lambda-connector.md

Updated 4.3.1: AWS_Region is no longer a Enum provided by lambda connector so I removed that part. 4.3.1 is now used to tabulate the ENUM_InvocationType.
